File: 3194-FindWordsContainingCharacter/3194-FindWordsContainingCharacter.js
// Last updated: 3/6/2026, 2:08:24 PM
/**
 * @param {string[]} words
 * @param {character} x
 * @return {number[]}
 */
var findWordsContaining = function(words, x) {
    let res = [];
    for(let i=0;i<words.length;i++){
        if(words[i].includes(x)){
            res.push(i);
        }
    }
    return res;
};

File: 3194-FindWordsContainingCharacter/3194-FindWordsContainingCharacter.cpp
// Last updated: 7/17/2026, 4:24:07 PM
class Solution {
public:
    vector<int> findWordsContaining(vector<string>& words, char x) {
        vector<int> res;

        for(int i=0;i<words.size();i++){
            for(char ch : words[i]){
                if(ch==x){
                    res.push_back(i);
                    break;
                }
            }
        }
        return res;
    }
};